pom.xml

```diff
@@ -288,6 +288,7 @@
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
+						<Main-Class>org.yaml.snakeyaml.Yaml2Yaml</Main-Class>
```

Comment:
Why the demo because the main class ?

Reply:
Convenience, so that people can use it out of the box like `java -jar snakeyaml.jar  file.yml`. E.g. I use it to create the config file for the [snmp-exporter configurator](https://github.com/jelmd/snmp_exporter/tree/jel) with the attached yaml file (a real world example): 
[generator.cisco.yml.txt](https://github.com/snakeyaml/snakeyaml/files/8350523/generator.cisco.yml.txt)